File: Taller del preparcial/Punto2.py
Numero1 = int(input("Introduzca el primero número: "))
Numero2 = int(input("Introduzca el segundo número: "))
Numero3 = int(input("Introduzca el tercer número: "))

if Numero1 > Numero2 or Numero1 > Numero3:
    print(Numero1)
elif Numero2 > Numero3:
    print(Numero2)
else: 
    print(Numero3)

File: Taller del preparcial/Punto2.cpp
#include <iostream>

using namespace std;

int main() {
int Numero1,Numero2,Numero3;

cout << "Introduce el primer numero: ";
cin >> Numero1;

cout << "Introduce el segundo numero: ";
cin >> Numero2;

cout << "Introduce el tercer numero: ";
cin >> Numero3;

if ( Numero1 > Numero2 && Numero1 > Numero3){

cout << "El número mayor es " << Numero1;

}
else if (Numero2 > Numero3 ){

cout << "El número mayor es " << Numero2;

}

else {

cout << "El número mayor es " << Numero3;


}




}